### ISSUE TITLE
Button About Issue 

### ISSUE THREAD

**Issue by eparascho** [algogr/HourglassQml#1]
Pressing the About button does not display any results to the user. 
I tried running the application from both Windows and Linux OS.
The result was the same.

https://user-images.githubusercontent.com/56228687/105350820-6172e680-5bf4-11eb-8784-03f76e20266d.png

**Comment by algogr** [comment]
Yes this is true. It does nothing and it is a bug. Would you like helping to resolve?

**Comment by eparascho** [comment]
I'm so sorry but I will not be able to participate in the bug fix due to lack of time and experience.
